.github/workflows/continuous-integration-workflow.yml

```diff
@@ -115,9 +124,12 @@ jobs:
           cmake --build builddir --parallel 2
           ccache -s
       - name: Tests
+        env:
+          ASAN_OPTIONS: allocator_may_return_null=1
```

Comment:
What is this option needed for?

Reply:
We intentionally allocate more memory than reasonably available to check the error message in some tests. It seemed reasonable to not care too much about this particular check.

Reply:
See [`09f12c6` (#6676)](https://github.com/kokkos/kokkos/pull/6676/commits/09f12c6983657b606489b5a824918a8c02e71c3d).

Reply:
Why did you not use an attribute?
https://clang.llvm.org/docs/AddressSanitizer.html#disabling-instrumentation-with-attribute-no-sanitize-address

Reply:
I'm not quite sure how exactly I would use it for just this unit test and disabling the unit test seemed good enough.

Reply:
Looks like the attempt in 335f7f2  for using attributes doesn't really work.

Reply:
I never managed to get it to work when I tried on my Mac a while ago.  Thanks for trying anyway.